docs/pages/concepts/logical/designs.md

```diff
@@ -14,9 +14,37 @@ Like a Google Doc, Designs are your primary tool for collaborative authorship of
 
 <!-- ### Using Designs -->
 
+A Design consists of **Components** and/or **Patterns**. A Design is the deployable unit in Meshery. Designs are how the users can describe the desired infrastructure state.
+There cannot be two components with the same name within a Design. However, there can be two components with the same name in different Designs.
```

Comment:
The names of the components are unique in a Design.

Reply:
Simpler. Easier o understand. 🙂